.github/workflows/build_cmake.yaml

```
@@ -81,6 +81,18 @@ jobs:
             sudo update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-14 100
           fi
 
+      - name: Setup MSVC dev command prompt
+        if: ${{ matrix.name == 'Windows MSVC' }}
+        run: ./.github/actions/setup-msvc
+        with:
```

actionlint (1.7.7)

this step is for running shell command since it contains at least one of "run", "shell" keys, but also contains "with" key which is used for running action

(syntax-check)